### ISSUE TITLE
Error on startup habitrpg-status

### ISSUE THREAD

**Issue by gabeos** [ryjm/habitrpg.el#18]
Thanks so much for this package. have been working on getting it set up under spacemacs, and am running into an internal wrong-type-argument error. debug stack trace is in gist below. package is loaded via 

```elisp
    (use-package habitrpg
      :init
      (progn
        (setq habitrpg-api-user "<redacted>")
        (setq habitrpg-api-token "<redacted>")
        (global-set-key (kbd "<f9> a") 'habitrpg-status))

      :config
      (progn
        (add-hook 'org-after-todo-state-change-hook 'habitrpg-add 'append)))
```

https://gist.github.com/gabeos/0548d4b23deba6b2b75abc23b5cffc2e

If you have any thoughts on how to proceed to get it set up, i'd appreciate any help.

**Comment by ryjm** [comment]
I haven't completely updated this to API V3 yet, so the status buffer wasn't working. My latest commit should get the status buffer close to working - there might still be some problems with adding/deleting tasks. Working on that now.

**Comment by gabeos** [comment]
Great, thanks so much! Looking forward to it.


On Thu, Aug 11, 2016, at 09:16 PM, ryjm wrote:
> Closed #18[1].
> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[2], or mute the thread[3].
>


Links:

  1. https://github.com/ryjm/habitrpg.el/issues/18
  2. https://github.com/ryjm/habitrpg.el/issues/18#event-753558872
